src/payment/cart/Cart.jsx

```diff
@@ -56,72 +56,72 @@ class Cart extends React.Component {
         </span>
 
         <CartContents>
-          {products && products.length ? products.map(product => (
+          {products.map(product => (
             <ProductLineItem
               key={product.title}
               isNumEnrolledExperiment={isNumEnrolledExperiment}
               REV1045Experiment={REV1045Experiment}
               isPaymentVisualExperiment={isPaymentVisualExperiment}
               enrollmentCountData={enrollmentCountData}
               {...product}
-            />)) : (REV1045Experiment ?
```

Comment:
Removing this experiment per @julianajlk feedback. Nested ternary operators violate new lint rules. So it was refactor or delete. Chose to delete